src/query/service/tests/it/sessions/session_context.rs

```diff
@@ -23,7 +23,10 @@ use databend_query::sessions::SessionContext;
 
 #[tokio::test]
 async fn test_session_context() -> Result<()> {
-    let conf = Config::load()?;
+    let conf = Config::load_with_configs(vec![(
```

Comment:
If the load only affects this test, I think we can not call `conf.load` for it, build the Config as:
```
let conf = Config {....}
```